src/arrays/arraymath.jl

```diff
@@ -143,12 +143,12 @@ Base.dot{B<:OrthonormalBasis}(qm1::AbstractQuMatrix{B}, qm2::DualMatrix{B}) = tr
 # normalization
 Base.norm(qarr::AbstractQuArray) = vecnorm(rawcoeffs(qarr))
 
-function normalize!(qarr::AbstractQuArray)
+function Compat.normalize!(qarr::AbstractQuArray)
```

Comment:
Yup it is mentioned in the README of the package https://github.com/JuliaLang/Compat.jl